Tools/PIDML/pidOnnxModel.h

```diff
@@ -213,48 +216,104 @@ struct PidONNXModel {
     }
   }
 
+  template <typename... P1, typename... P2>
+  static constexpr bool is_equal_size(o2::framework::pack<P1...>, o2::framework::pack<P2...>)
+  {
+    return sizeof...(P1) == sizeof...(P2);
+  }
+
+  static float scale(float value, const std::pair<float, float>& scalingParams)
+  {
+    return (value - scalingParams.first) / scalingParams.second;
+  }
+
+  template <typename C, typename T>
+  typename C::type getPersistentValue(const T& rowIterator)
+  {
+    return *(static_cast<C>(rowIterator).getIterator());
+  }
+
+  template <typename T, typename... C>
+  std::optional<float> getColumnValueByLabel(o2::framework::pack<C...>, const T& rowIterator, const std::string& columnLabel)
```

Comment:
@aalkin @ktf 
Perhaps we could have this function in O2 instead of O2Physics?

Reply:
Then it would be `type name C::type` as the return type; for ML usage, hardcoded float cast was reasonable - but if it would be in O2, then not anymore.

Reply:
> @aalkin @ktf Perhaps we could have this function in O2 instead of O2Physics?

In line of principle, I do not see why not. Just open a separate PR and we discuss the details there.